### ISSUE TITLE
analyze files in src directory before starting REPLs

### ISSUE THREAD

**Issue by swannodette** [emezeske/lein-cljsbuild#76]
One common problem is that REPLs start without any information about what is loaded into the evaluation environment (the browser, etc). An easy way to make the REPL more usable and to avoid a lot of bugs around namespace functionality (like in-ns), would be to analyze all files in the source path before starting the REPLs.

**Comment by cemerick** [comment]
AFAICT, this is no longer relevant post-new-REPL stuff in CLJS upstream.
